### PR TITLE
Renamed ContainedButton et al. to ElevatedButton et al.

### PR DESCRIPTION
**Pull request by HansMuller**
The "New Button Universe" PR, https://github.com/flutter/flutter/pull/59702, landed less than 24 hours ago. We've just been informed by Material Design spec developers that they'd _really_ prefer to call `ContainedButton` `ElevatedButton`. In fact, when an updated Material Design spec is published, the official name will have been changed. 

Since the ContainedButton class is so new, and because "ElevatedButton" is a little closer to the original "RaisedButton" name, we're going to quickly rename ContainedButton to be ElevatedButton now, and deprecate the old class names. The old classes still exist, but should be gone within a week.

I apologize for the confusion.

### CODE REVIEW

File: packages/flutter/lib/src/material/elevated_button.dart [hunk 1]
```diff
@@ -369,6 +369,138 @@ class _ContainedButtonDefaultMouseCursor extends MaterialStateProperty<MouseCurs
...
+    return Row(
+      mainAxisSize: MainAxisSize.min,
+      children: <Widget>[icon, SizedBox(width: gap), label],
+    );
+  }
+}
+
+@Deprecated(
+  'This class was briefly released with the wrong name. The correct name is ElevatedButton. '
+  'This class was deprecated after within 24 hours of its debut'
```
Comment (darrenaustin) [inline packages/flutter/lib/src/material/elevated_button.dart:429]: 'was deprecated after 24 hours of its debut '

**Review by darrenaustin — APPROVED** [review]
Minor wording change, but LGTM.

File: packages/flutter/lib/src/material/elevated_button_theme.dart [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+  @override
+  bool updateShouldNotify(ElevatedButtonTheme oldWidget) => data != oldWidget.data;
+}
+
+
+/// Please use [ElevatedButtonTheme].
+@Deprecated(
+  'This class was briefly released with the wrong name. '
+  'The correct name is ContainedButtonThemeData. '
```
Comment (Levi-Lesches) [inline packages/flutter/lib/src/material/elevated_button_theme.dart:133]: Shouldn't this be `ElevatedButtonThemeData`?

**Comment by Levi-Lesches** [comment]
In /lib/src/material/elevated_button_theme.dart, line 133 has the wrong name. 

https://github.com/flutter/flutter/pull/61262/files#r453116191

File: packages/flutter/lib/src/material/elevated_button_theme.dart (hunk 2 above)
Comment (HansMuller) [reply packages/flutter/lib/src/material/elevated_button_theme.dart]: Yes, you're correct, thanks for pointing that out.

**Pull request merged**